src/main/java/com/dubreuia/core/component/Engine.java

```diff
@@ -77,9 +81,12 @@
         this.mode = mode;
     }
 
-    void processPsiFilesIfNecessary() {
+    public void processPsiFilesIfNecessary() {
+        if (psiFiles == null) {
+            return;
+        }
         if (!storage.isEnabled(activation)) {
-            LOGGER.info("Plugin not activated on " + project);
+            LOGGER.info("Action \"" + activation.getText() + "\" not enabled on " + project);
```

Comment:
Can we use `String.format()` here too as like in other places?

Reply:
@abimarank Thanks for reviewing. I replace all string concats in Engine class by String.format.